### ISSUE TITLE
renderer options maxCanvasWidth & maxCanvasHeight not respected when embedded

### ISSUE THREAD

**Issue by s-light** [aframevr/aframe#4911]
## Description

the [renderer options `maxCanvasWidth`  & `maxCanvasHeight`](https://aframe.io/docs/1.2.0/components/renderer.html#properties_maxcanvaswidth) are not respected when the [`embedded`](https://aframe.io/docs/1.2.0/components/embedded.html) option is set.

- A-Frame Version: 1.2.0
- Platform / Device: 
```
Name                Firefox
Version             91.0.1
Build ID            20210816143654
Distribution ID     canonical
User Agent          Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:91.0) Gecko/20100101 Firefox/91.0
OS                  Linux 5.4.0-81-generic #91-Ubuntu SMP Thu Jul 15 19:09:17 UTC 2021
```
- Reproducible Code Snippet or URL:
  - [working jsfiddle](https://jsfiddle.net/s_light/19bnokae/37/)
  - [broken jsfiddle](https://jsfiddle.net/s_light/19bnokae/39/)
```
<a-scene
      stats
      embedded
      renderer="
            maxCanvasWidth: 256;
            maxCanvasHeight: 256
            "
    >
      ....
    </a-scene>
```

**Comment by s-light** [comment]
found the relevant code lines:
- [`resize`](https://github.com/aframevr/aframe/blob/master/src/core/scene/a-scene.js#L579) is calling [`getCanvasSize()`](https://github.com/aframevr/aframe/blob/master/src/core/scene/a-scene.js#L579)
- and in [`getCanvasSize()`](https://github.com/aframevr/aframe/blob/master/src/core/scene/a-scene.js#L818) there is no check for the maxCanvasSize if embedded is active:
```js
function getCanvasSize (canvasEl, embedded, maxSize, isVR) {
  if (!canvasEl.parentElement) { return {height: 0, width: 0}; }
  if (embedded) {
    return {
      height: canvasEl.parentElement.offsetHeight,
      width: canvasEl.parentElement.offsetWidth
    };
  }
  return getMaxSize(maxSize, isVR);
}
```

currently i have not a clear idea how to fix this..

**Comment by s-light** [comment]
fix in #4912 
test case:
https://codepen.io/s-light/pen/NWgWoJm?editors=1001
and the original jsfiddle linked against the fix:
https://jsfiddle.net/s_light/1zwvmh0e/1/
